src/InfoViz/Native/HistogramSelector/index.js

```diff
@@ -843,6 +819,8 @@ const DEFAULT_VALUES = {
   defaultScore: 0,
   dragMargin: 8,
   selectedDef: null,
+
+  numberOfBins: 32,
```

Comment:
Not with the new provider, as you can subscribe to various number of bins across components.
So this should be an application problem, not a provider one to ensure components use the same number of bins when they get created.